client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -918,10 +927,14 @@ const DataManagementRow = ({
                                 </Button>
                             )}
                             <TestPlanReportStatusDialogWithButton
+                                role="listitem"
                                 testPlanVersionId={latestVersion.id}
                             />
-                            <span className="more">
-                                <span className="more-issues-container">
+                            <span role="list" className="more">
+                                <span
+                                    role="listitem"
+                                    className="more-issues-container"
+                                >
```

Comment:
This definitely sounds better and it broken up into fewer parts. The only part that was challenging on VoiceOver was when I first focus, I only hear "List 2 Items, Level 2". These two items are connected to one another visually for the sake of the layout but it doesn't feel like they are conceptually linked. If this is the case, could they be separated out of a list so that they read separately without the additional layer of having the list read? If they are conceptually linked, maybe the list element could have a label that described the combination of listitems?